qa/metrics.py

```diff
@@ -0,0 +1,84 @@
+from abc import ABC, abstractmethod
+
+class QAMetric(ABC):
```

Comment:
This could also be written as a dataclass which might be a bit more clear:

then metrics would be instantiated like:
```
HospitalBedsRequired = QAMetric(
    name="HospitalBedsRequired",
    projection_path=["timeseries", "hospitalBedsRequired"],
    actual_path=None,
    threshold=1
)
```

Reply:
i think that the ABC makes it so that you could overwrite the diff/ compare function which is kind of nice